wolfssl/openssl/evp.h

```diff
@@ -56,6 +56,7 @@
 #endif
 
 typedef char WOLFSSL_EVP_CIPHER;
+typedef char WOLFSSL_EVP_MD;
```

Comment:
Removed duplicate definitions.
I just bought a Mac Book Pro, so, will be able to do this in future.